cmd/kube-apiserver/app/options/options.go

```diff
@@ -78,6 +78,8 @@ type ServerRunOptions struct {
 	ServiceAccountSigningKeyFile     string
 	ServiceAccountIssuer             serviceaccount.TokenGenerator
 	ServiceAccountTokenMaxExpiration time.Duration
+
+	ShowHiddenMetrics string
```

Comment:
Either add a comment or name it "ShowHiddenMetricsForVersion"?

Reply:
Done.